Sources/IssueReporting/Documentation.docc/Articles/GettingStarted.md

```diff
@@ -20,7 +20,11 @@ else {
 By default, this will trigger an unobtrusive, purple runtime warning when running your app in Xcode
 (simulator and device):
 
-![A purple runtime warning in Xcode showing that an issue has been reported.](runtime-warning)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="../Resources/runtime-warning~dark.png">
+  <source media="(prefers-color-scheme: light)" srcset="../Resources/runtime-warning.png">
+  <img alt="A purple runtime warning in Xcode showing that an issue has been reported." src="../Resources/runtime-warning.png">
+</picture>
```

Comment:
Hi @Mika5652, I don't believe these changes are necessary. Since this is a DocC markdown file we get use some of their specific markdown syntax, which will automatically figure out the dark/light image from the base image name, `runtime-warning`. If you undo the changes in the DocC files I will merge this.

Reply:
Hi, I am ok with that, but when I am looking at https://github.com/pointfreeco/swift-issue-reporting/blob/main/Sources/IssueReporting/Documentation.docc/IssueReporting.md or the `GettingStarted` file I don't see images but question marks instead. I thought that wasn't correct. So maybe we should use markdown syntax but with the correct path. So this `![A purple runtime warning in Xcode showing that an issue has been reported.](../Resources/runtime-warning.png)` ?

Reply:
Unfortunately DocC does not allow for paths or extensions in the resource. It really does have to be the bare `runtime-warning`.

I think we just have to consider DocC a format that is mostly meant to be rendered, not viewed in its raw form.

Reply:
I see. I leave there only changes in README if those are ok. Thanks for the explanation!